rubin_scheduler/utils/consdb.py

```diff
@@ -316,8 +316,32 @@ def consdb_visits(self) -> pd.DataFrame:
         prior_day_obs_date = day_obs_date - timedelta(days=self.num_nights)
         prior_day_obs_int = int(prior_day_obs_date.strftime("%Y%m%d"))
 
+        # To avoid duplicate columns, explicitly include list of columns to
+        # to return in the query instead of using a "SELECT *".
+        def columns_query(table, instrument):
+            return f"""
+                SELECT column_name FROM information_schema.columns
```

Comment:
Based on KT's reaction, might want to think about whether information_schema.columns is the way to go or whether it should be the ConsDB client schema(schema, table) call.  (it would however be interesting to check that you get the same values .. I see that some keys are missing from the values returned by that call). 
We may want to move into more consDB client usage as well, instead of raw postgres or sqlachemy .. the consdb client is intended to have some flex with changing columns I think (through some of its "flexible metadata" use), so may isolate a bit from changes in the column names due to changes in the butler. 
I've been ok using the ConsDB client directly.

Reply:
I used information_schema.columns directly because that's what I already knew how to do; for our use, using the ConsDB client seems like learning and extra new API that doesn't really add anything I can't already do just as easily with raw postgresql.
I see some trade-offs in stability, and it's not clear which "wins" here. As you point out, maybe there will be fiddles in the ConsDB client that insulate code from schema changes in the database. On the other hand, querying  information_schema.columns is guaranteed to be correct (because that's how postgresql itself knows what columns are in the table), while it's possible for the EFDClient to be wrong about its schema (I think this has already happened), and using EFDClient also means that the code is vulnerable to changes in the EFDClient module itself, while direct postgresql queries are not.

Reply:
Hm. Ok.